### PR TITLE
Add ttl property to annotation document

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**
* Add ttl to Document Annotation attribution

* Add UI and IT for TTL.

Signed-off-by: Pan Li <panli@microsoft.com>

### CODE REVIEW

File: src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java [hunk 1]
```diff
@@ -80,5 +89,27 @@ public void testDocumentDBAnnotationIT() {
         TestUtils.testIndexingPolicyPathsEquals(policy.getIncludedPaths(), TestConstants.INCLUDEDPATHS);
         TestUtils.testIndexingPolicyPathsEquals(policy.getExcludedPaths(), TestConstants.EXCLUDEDPATHS);
     }
+
+    @Test
+    @SneakyThrows
```
Comment (sophiaso) [inline src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java]: Why we use `SneakyThrows` here?

Comment (Incarnation-p-lee) [reply src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java]: TimeUnit will throw exception, and we'd like not to handle that here.

Comment (sophiaso) [reply src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java]: Why not explicitly use `throws` in the method signature?

Comment (Incarnation-p-lee) [reply src/test/java/com/microsoft/azure/spring/data/documentdb/repository/DocumentDBAnnotationIT.java]: I think that may looks simple and clean for test code.

**Pull request merged**